### DISCUSSION TITLE
npm edge-js OLEDB package

### DISCUSSION THREAD

**Discussion by mshsun20** [npm/feedback#1032] [Documentation] [answered]
npm edge-js OLEDB package technical process details. how to implement connection between Node.js custom Apps and OLEDB, so that data processing could be possible in between these. Your assistance is very much required as there are very little details found online regarding this.

**Comment by MylesBorins** [comment, marked as answer]
This feedback repository is for providing direct feedback about npm itself, not for support with specific packages. Please try github.community or stack overflow
